Winging it 10

SondraE opened the pull request:
## Related Issue(s)
[#676](https://github.com/oddbird/oddleventy/issues/676)


## Steps to test/reproduce
posts list
/2024/07/23/winging-it-10/

mirisuzanne commented on content/blog/wingingit/winging-it-10.md:
```diff
@@ -0,0 +1,65 @@
+  - youtube: aDMWD_CYpEI
+summary: |
+  We talk with Stephanie Eckles
+  – the brain behind ModernCSS.dev and SmolCSS.css
+  – about CSS grids.
```
```suggestion
  -- the brain behind ModernCSS.dev and SmolCSS.css
  -- about CSS grids.
```

Since it's hard to tell the difference in monospace fonts, we generally let 'typogr' create dashes for us.

mirisuzanne commented on content/blog/wingingit/winging-it-10.md:
```diff
@@ -0,0 +1,65 @@
+  We talk with Stephanie Eckles
+  – the brain behind ModernCSS.dev and SmolCSS.css
+  – about CSS grids.
+  We'll look at a few of the most common grid patterns,
```
```suggestion
  We look at a few of the most common grid patterns,
```

The tense isn't consistent

mirisuzanne reviewed (approved):
Generally looks good, just some cleanup in the summary.

stacyk commented on content/blog/wingingit/winging-it-10.md:
```diff
@@ -0,0 +1,65 @@
+  - youtube: aDMWD_CYpEI
+summary: |
+  We talk with Stephanie Eckles
+  – the brain behind ModernCSS.dev and SmolCSS.css
```
Feels like we should link these, but if we can't/shouldn't then at minimum the address is SmolCSS.dev (not .css). I'll check to see if the YouTube description has this same typo? 

```suggestion
  – the brain behind [ModernCSS.dev](https://moderncss.dev/) and [SmolCSS.dev](https://smolcss.dev/)
```

SondraE commented:
@stacyk @mirisuzanne I've addressed all your comments. This has the same video thumbnail issue as the other videos. I assume that should not hold up this PR, right?

mirisuzanne reviewed (approved):
lgtm
